entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -781,3 +781,80 @@ This change was made so that the version of SQLite used on iOS consistent with o
 **Mitigations**
 
 To use the native SQLite version on iOS, configure `Microsoft.Data.Sqlite` to use a different `SQLitePCLRaw` bundle.
+## Char values are now stored as TEXT on SQLite
+
+[Tracking Issue #15020](https://github.com/aspnet/EntityFrameworkCore/issues/15020)
+
+This change was introduced in EF Core 3.0-preview 4.
+
+**Old behavior**
+
+Char values were previously sored as INTEGER values on SQLite. For example, a char value of *A* was stored as the integer value 65.
+
+**New behavior**
+
+Char values are now sotred as TEXT.
+
+**Why**
+
+Storing the values as TEXT is more natural and makes the database more compatible with other technologies.
+
+**Mitigations**
+
+You can migrate existing databases to the new format by executing SQL like the following.
+
+``` sql
+UPDATE MyTable
+SET CharColumn = char(CharColumn)
+WHERE typeof(CharColumn) = 'integer';
+```
+
+You could also continue using the previous behavior by configuirng a value converter on your model properties.
```

Comment:
Do you want to say "if you are using EF Core"?

Reply:
Possibly. So far, this page is all about EF Core. We haven’t added the ADO.NET-only SQLite breaking changes.